supplementary/alica_ros1/alica_ros_turtlesim/libalica-turtlesim/src/Teleport.cpp

```diff
@@ -13,14 +12,23 @@ Teleport::Teleport(alica::BehaviourContext& context)
 {
 }
 
+void Teleport::run()
+{
+    if (isSuccess() || isFailure()) {
+        return;
+    }
+    // wait for turtle to have a valid position
+    if (_turtle->getCurrentPose()) {
```

Comment:
We should probably also wait until the pose is the one we actually teleported to.  Otherwise we might use an outdated pose for utility